src/Layout.js

```diff
@@ -1122,8 +1122,50 @@ var computeLayout = (function() {
       child.nextAbsoluteChild = null;
     }
   }
+  
+  function layoutNode(node, parentMaxWidth, parentDirection) {
+    if (node.isDirty ||
```

Comment:
Can you write this as a local variable skipLayout the same way the C version does it?